zeeguu/core/model/article.py

```diff
@@ -22,6 +22,7 @@
 )
 
 MAX_CHAR_COUNT_IN_SUMMARY = 300
+MARKED_BROKEN_DUE_TO_LOW_QUALITY = 100
```

Comment:
sorry Tiago, but we don't seem to be using this constant anywhere. Or am I missing something?